### PR TITLE
add metadata to keep pages

### PR DESCRIPTION
**Pull request by camhashemi**
@andrewconner @eishay

### CODE REVIEW

File: kifi-backend/modules/shoebox/app/com/keepit/commanders/PageMetaTagsCommander.scala [hunk 1]
```diff
@@ -51,6 +52,7 @@ class PageMetaTagsCommander @Inject() (
     libraryRepo: LibraryRepo,
     libraryMembershipRepo: LibraryMembershipRepo,
     rover: RoverServiceClient,
+    implicit val imageStore: S3ExternalIdImageStore,
```
Comment (andrewconner) [inline kifi-backend/modules/shoebox/app/com/keepit/commanders/PageMetaTagsCommander.scala]: I think I saw a PR where this was removed from @leogrim

File: kifi-backend/modules/shoebox/app/com/keepit/commanders/PageMetaTagsCommander.scala [hunk 2]
```diff
@@ -255,24 +254,37 @@ class PageMetaTagsCommander @Inject() (
...
+        val splitName = author.name.split(" ")
+        (splitName.headOption.getOrElse(""), splitName.lastOption.getOrElse(""))
+      }
+      PublicPageMetaFullTags(
         unsafeTitle = keep.title.getOrElse(""),
         url = url,
         urlPathOnly = urlPath,
         feedName = None,
         unsafeDescription = keep.note.orElse(keep.title).getOrElse(""),
-        images = Seq(), // todo(cam): get this from embeddly
```
Comment (andrewconner) [inline kifi-backend/modules/shoebox/app/com/keepit/commanders/PageMetaTagsCommander.scala:268]: :rocket:

**Comment by andrewconner** [comment]
Seems reasonable

File: kifi-backend/modules/shoebox/app/com/keepit/commanders/PageMetaTagsCommander.scala (hunk 1 above)
Comment (camhashemi) [inline kifi-backend/modules/shoebox/app/com/keepit/commanders/PageMetaTagsCommander.scala]: indeed, merge conflicts

**Pull request merged**